ckanext/restricted/tests/test_performance_package_search.py

```diff
@@ -0,0 +1,117 @@
+# encoding: utf-8
+import time
+import os.path
+
+from ckan.tests import helpers
+import logging
+import pytest
+from assertpy import assert_that
+from ckan.common import config
+import subprocess
+
+import ckanext.restricted.action
+import ckanext.restricted.plugin
+import ckanext.restricted.tests.old_action
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def import_performance_data(clean_db, clean_index):
+    raw_db_url = config['sqlalchemy.url']
+    ckan_dir = get_ckan_directory()
+    sql_file = f'{ckan_dir}/ckanext-restricted/ckanext/restricted/tests/performance_test_data.sql'
+    cmd = ['psql',  f'{raw_db_url}', "-f", f'{sql_file}', '> /dev/null']
+
+    log.info(f"Loading performance data using: '{cmd}'")
+    subprocess.run(cmd)
```

Comment:
Maybe this is only for me but when I've run the tests with `adx test restricted` my logs got very polluted with sql INFO statments like:
```bash
[...]
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
INSERT 0 1
 setval 
--------
      1
(1 row)
```

Reply:
Yep, got rid of it.